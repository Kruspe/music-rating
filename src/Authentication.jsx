import React from 'react';
import { Authenticator } from 'aws-amplify-react';
import Amplify from 'aws-amplify';
import App from './App';

import awsExports from './aws-exports';
import './App.css';

Amplify.configure(awsExports);

const Authentication = () => (
  <Authenticator>
    <App />
  </Authenticator>
);

export default Authentication;
