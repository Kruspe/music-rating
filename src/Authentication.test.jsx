import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Authentication from './Authentication';

describe('Authentication', () => {
  it('should render correctly', () => {
    expect(toJSON(shallow(<Authentication />))).toMatchSnapshot();
  });
});
