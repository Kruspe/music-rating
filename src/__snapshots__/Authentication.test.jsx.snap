// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authentication should render correctly 1`] = `
<Authenticator>
  <App />
</Authenticator>
`;
